packages/labs/src/components/select/select.tsx

```diff
@@ -114,9 +114,18 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
 
     public state: ISelectState<T> = { isOpen: false, query: "" };
 
+    private input: HTMLInputElement;
     private TypedQueryList = QueryList.ofType<T>();
     private list: QueryList<T>;
     private refHandlers = {
+        input: (ref: HTMLInputElement) => {
+            this.input = ref;
+
+            const { inputProps = {} } = this.props;
+            if (inputProps.inputRef !== undefined) {
```

Comment:
Can use `safeInvoke`. Looks like we already import core `Utils` into this file, but this is how I prefer to do it in `packages/table` at least, given the overloaded `utils.ts` filename that appears in both `core` and `table`:

```tsx
import { Utils as CoreUtils } from "@blueprintjs/core";
// ...
const { inputProps = {} } = this.props;
CoreUtils.safeInvoke(inutProps.inputRef, ref);
```